css/src/components/messages.scss

```diff
@@ -0,0 +1,58 @@
+.messages {
+	display: flex;
+	flex-direction: column;
+	max-width: 100%;
+	gap: $layout-4;
+}
+
+.chat-bubble {
+	padding: $spacer-4;
+	border: $border-width solid $border !important;
+	border-radius: $border-radius !important;
+	box-shadow: $box-shadow-light !important;
+}
+
+.chat-bubble,
+.message-time {
+	width: 80%;
+}
+
+.message-time,
+.user-title {
+	color: $text-subtle;
+	font-size: $font-size-8;
+}
+
+.chat-header,
+.message-time {
+	margin-block-end: $spacer-4;
+}
+
+.chat-title {
+	display: flex;
+
+	p {
```

Comment:
We should  not have any bare element selectors

Reply:
updated